specs-go/config.go

```diff
@@ -126,7 +126,7 @@ type Hook struct {
 	Path    string   `json:"path"`
 	Args    []string `json:"args,omitempty"`
 	Env     []string `json:"env,omitempty"`
-	Timeout *int     `json:"timeout,omitempty"`
```

Comment:
In the interest of landing the normative spec change, I've dropped the de-pointerization here with 8510055 → ecf7314 and opened #772 around whether we need invalid-zero-value detection for optional parameters.